dashboard/src/components/InstanceView/InstanceView.tsx

```diff
@@ -205,7 +210,7 @@ export class InstanceView extends React.Component<IInstanceViewProps> {
             </CardGrid>
             <h2>Bindings</h2>
             <AddBindingButton
-              bindingName={instance.metadata.name + "-binding"}
+              bindingSchema={svcPlan && svcPlan.spec.serviceBindingCreateParameterSchema}
```

Comment:
perhaps you're misunderstanding the issue in #363, @arapulido wasn't able to create the bindings for the GCP PubSub service because there were required parameters and previously this form didn't support specifying parameters for creating bindings (we only supported them for provisioning instances). So this PR is to add support for the schema provided by the broker to render a form for entering binding parameters.

Reply:
I see now, I didn't realized how pieces are tied together